lib/plugins/aws/deploy/compile/events/apiGateway/lib/authorizers.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const _ = require('lodash');
+
+module.exports = {
+  compileAuthorizers() {
+    _.forEach(this.serverless.service.functions, (functionObject, functionName) => {
+      functionObject.events.forEach(event => {
```

Comment:
oh yeah...I thought we should do that once we merge the other PR. Or better yet, after we merge this, the other PR could be updated.